README.md

```diff
@@ -1,7 +1,7 @@
 <h2 align="center">
   <a href="http://mlpack.org"><img
 src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="background-color:rgba(0,0,0,0);" height=230 alt="mlpack: a fast, flexible machine learning library"></a>
-  <br>a fast, flexible machine learning library<br>
+  <br>a fast, flexible header-only machine learning library<br>
```

Comment:
@shrit maybe simplify to "header-only machine learning library".

too many qualifiers can muddle the main message

Reply:
Yeah, that seems fine to me, `fast, header-only` gets the idea across still, in my opinion. :+1:

Reply:
Indeed that can work as well